app/models/data_mapper_setup.rb

```diff
@@ -0,0 +1,9 @@
+require 'data_mapper'
+require 'dm-postgres-adapter'
+require './app/models/user'
+require './app/models/peep'
+
+# DataMapper::Logger.new($stdout, :debug)
+DataMapper.setup(:default, ENV['DATABASE_URL'] || "postgres://localhost//chitter_#{ENV['RACK_ENV']}")
```

Comment:
Metrics/LineLength: Line is too long. [101/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)